### PR TITLE
New home and hub initial

### PR DESCRIPTION
**Pull request by recrwplay**

### CODE REVIEW

File: preview-src/docs-sub-hub.adoc [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+Lorem ipsum dolor sit est.
+
+
+. *Tutorial: Getting started with Cypher*
++ 
+Learn stuff
+. *Working with data sets*
++
+Learn stuff
+. *Step 3 goes here*
```
Comment (lidiazuin) [inline preview-src/docs-sub-hub.adoc:87]: Question: with these changes, now every time we use `.` it will turn into the blue circled number?

Comment (recrwplay) [reply preview-src/docs-sub-hub.adoc:87]: No, it's using a role that is applied to the whole list, in this case `[.next-steps]`, so only lists with that role would have the styles applied.

Also, the css styles for this role are only applied when the `page-layout` attribute is set to `docs-ndl`, which we will use for all the new index pages. If we want to use this style of list anywhere else we could remove that requirement.

Comment (lidiazuin) [reply preview-src/docs-sub-hub.adoc:87]: True, I missed the `[.next-steps]` line. I was wondering if that could be helpful for tutorial pages at some point, but we can revisit that later. Thanks for explaining!

Comment (recrwplay) [reply preview-src/docs-sub-hub.adoc:87]: Yes, this could become a reusable pattern can apply to existing docs - tutorials are a good example

Comment (lidiazuin) [reply preview-src/docs-sub-hub.adoc:87]: We are missing the feedback widget. I guess we can use the same style, just update with NDL colors?

Comment (recrwplay) [reply preview-src/docs-sub-hub.adoc:87]: I've restored the feedback widget - it disabled it at some point in the pages while developing

Comment (recrwplay) [reply preview-src/docs-sub-hub.adoc:87]: I've added a `border-radius` to display slightly rounded corners

File: src/css/docs-ndl.css [hunk 2]
```diff
@@ -332,41 +332,19 @@ body.docs-ndl .lists .sect2 {
...
-  height: 4rem;
-  width: 4rem;
-} */
-
-body.docs-ndl.explainer .cards .sect2 a h3,
-body.docs-ndl.explainer .cards .sect2 a .description {
-  margin-left: 5rem;
 }
 
 body.docs-ndl.color-docs .cards .sect2 h3,
```
Comment (lidiazuin) [inline src/css/docs-ndl.css:413]: In line 412, could we have something like this for the icons to be aligned to the left of the text? I suppose this is for the list widget that is being used in the Tutorials subpage? 

```
body.docs-ndl .lists .sect2 .icon img,
body.docs-ndl .icon-s .sect2 .icon img {
  width: 3rem;
  height: 3rem;
  padding: 0;
  margin: 40px;
  float: left
}
```

**Pull request merged**